lib/widgets/read_times_tamp.dart

```diff
@@ -9,9 +9,8 @@ import 'package:timeago/timeago.dart' as timeago;
 //https://pub.dev/documentation/timeago/latest/timeago/timeago-library.html
 
 String timesTampToTimeAgo(String timestamp) {
-
-  var covertTimesTamp = DateTime.parse(timestamp);
-  covertTimesTamp = covertTimesTamp.toUtc();
+  // Note: Timestamps always come as UTC but don't have the time zone set on EOSIO
+  var covertTimesTamp = DateTime.parse(timestamp + "Z");
```

Comment:
Why not:

` DateTime d = DateTime.parse(utc).toLocal();`

Reply:
Because that creates the wrong time. 

Parsing the date creates a epoch timestamp - milliseconds since epoch. This timestamp has nothing to do with time zones - time zones are only for display

The bug was the way we were interpreting the incoming string, we created the wrong epoch timestamp. 

Whatever follows that doesn't matter because the time has been parsed the wrong way

The time must be parsed initially as UTC, and the way to do that in flutter parser is to append a 'Z' for UTC timezone

Reply:
oh I see then the problem was not to set up the premises or the time ago, the problem was to get the utc from a timestamp.